### DISCUSSION TITLE
New Version for MathPlugin?

### DISCUSSION THREAD

**Discussion by Ianuaris** [typemill/typemill#473] [General]
Hey and thanks and congratulations. Great work!

I'm just questioning, if there will be an update for the Math-Plugin for Typemill v2.
I would like to use Typemill even in the future, but a working Markdown-Math-Support is essential for my use cases.

**Comment by trendschau** [comment]
Yes, it is on the roadmap, but I can't say right now when I find the time for it because the roadmap is pretty crowded....

**Comment by godixche** [comment]
I would like to see an updated Math-Plugin too. Maybe it is possebly that you also support the mhchem option in KaTeX.

**Comment by trendschau** [comment]
Yes, it is on the roadmap, but I have to find time for it since math plugin is a bit more sophisticated with the integration into the editor. This part has been refactored in Typemill v2 so I have to rewrite the old math plugin, I hope to find time next two months

**Comment by trendschau** [comment]
So I updated the math plugin and other old plugins now: https://plugins.typemill.net/math 

I did not add special features like the mhchem option and it would be great to find someone in the community to maintain this plugin since I don't have the time for it. If someone is interested, I can upload the latest version to github so you can fork and maintain.
